sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryUtils.java

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.tracing.opentelemetry;
+
+import com.azure.core.util.logging.ClientLogger;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.StatusCode;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static com.azure.core.util.tracing.Tracer.ENTITY_PATH_KEY;
+import static com.azure.core.util.tracing.Tracer.HOST_NAME_KEY;
+
+class OpenTelemetryUtils {
+    private static final ClientLogger LOGGER = new ClientLogger(OpenTelemetryUtils.class);
+
+    private static final Map<String, String> ATTRIBUTE_MAPPING_V1_12_0 = getMappingsV1200();
+    static final String SERVICE_REQUEST_ID_ATTRIBUTE = "serviceRequestId";
+    static final String CLIENT_REQUEST_ID_ATTRIBUTE = "requestId";
+
+    private static Map<String, String> getMappingsV1200() {
+        Map<String, String> mappings = new HashMap<>(4);
```

Comment:
You'll want to use a size larger than 4 as the Map will resize when more than 4 * LOAD_FACTOR (default is 0.75) entries are set